### ISSUE TITLE
sapphire-localnet: -amount broken

### ISSUE THREAD

**Issue by matevz** [oasisprotocol/oasis-web3-gateway#585]
##### SUMMARY

Reported by @csillag:

```
oa@matevz-oa:~/playground$ docker run -it -p8545:8545 -p8546:8546 -e OASIS_NODE_LOG_LEVEL=debug -e LOG__LEVEL=debug ghcr.io/oasisprotocol/sapphire-localnet -amount 10 -test-mnemonic
sapphire-localnet 2024-06-17-gitb807232 (oasis-core: 24.0-gitfb49717, sapphire-paratime: 0.7.3-testnet, oasis-web3-gateway: 5.1.0-gitb807232)

 * Starting oasis-net-runner with sapphire...
 * Waiting for Postgres to start.....
 * Waiting for Oasis node to start...
 * Starting oasis-web3-gateway...
 * Bootstrapping network (this might take a minute).
 * Waiting for nodes to be ready....
 * Setting epoch to 1...
 * Waiting for key manager to generate ephemeral secret....
 * Setting epoch to 2...
 * Waiting for key manager......
 * Populating accounts...

panic: can't deposit: client: transaction check failed: runtime error: module: consensus code: 5 message: consensus: amount not representable

goroutine 1 [running]:
main.main()
        /go/oasis-web3-gateway/docker/common/oasis-deposit/main.go:273 +0x1005
```

I think we need to bump the oasis-sdk Go bindings in the deposit helper.

**Comment by kostko** [comment]
Is the amount passed as an argument in base units and 10 is less than 1 consensus layer base unit so you cannot deposit such a small amount? Try increasing the amount ;)

**Comment by CedarMist** [comment]
So the intuition is people expect to pass whole units of ROSE to `amount` but it's actually base units...

**Comment by matevz** [comment]
Should be implicitly fixed by https://github.com/oasisprotocol/oasis-web3-gateway/issues/617.

**Comment by matevz** [comment]
Fixed in #633.
